docs/button.md

```diff
@@ -222,7 +222,7 @@ TV preferred focus.
 
 ---
 
-### `nextFocusDown` <div className="label android">Android</div><div className="label tv">TV</div>
+### `nextFocusDown` <div className="label android">Android TV</div>
```

Comment:
In the TV repo, this prop (and other similar props below) are supported on both Apple TV and Android TV.